lib/simulator-xcode-6.js

```diff
@@ -450,7 +450,7 @@ class SimulatorXcode6 extends EventEmitter {
    * @param {boolean} scrub - If `scrub` is false, we want to clean by deleting the app and all
    *   files associated with it. If `scrub` is true, we just want to delete the preferences and
    *   changed files.
-   * @return {array} Array of deletion promises.
+   * @return {array<Promise>} Array of deletion promises.
```

Comment:
This method ends in `await B.all(deletePromises);`. So it does not return anything at all.

Reply:
fixed